utils/_context/_scenarios/__init__.py

```diff
@@ -1,5 +1,6 @@
 import json
 
+from utils._context._scenarios.aws_lambda import LambdaScenario
```

Comment:
Could you mive this few line below with its sibling ? 

```suggestion
from .aws_lambda import LambdaScenario
```